test/cypress/integration/20-create-collectiveFromGithub.test.js

```diff
@@ -1,31 +1,6 @@
 describe('Create collective from Github', () => {
   it('Should submit create github page', () => {
-    cy.server();
-    cy.route({
-      method: 'GET',
-      url: '/api/github-repositories?*',
-      /* eslint-disable camelcase */
-      response: [
-        {
-          description: 'Adblock Plus browser extension',
-          fork: true,
-          full_name: 'testuseradmingithub/adblockpluschrome',
-          name: 'adblockpluschrome',
-          owner: { login: 'testuseradmingithub', type: 'Organization' },
-          stargazers_count: 113,
-        },
-        {
-          description:
-            'A new form of association, transparent by design. Please report issues there. Feature requests and ideas welcome!',
-          fork: true,
-          full_name: 'testuseradmingithub/jobtweets',
-          name: 'JobTweets',
-          owner: { login: 'testuseradmingithub', type: 'User' },
-          stargazers_count: 103,
-        },
-      ],
-      /* eslint-enable camelcase */
-    });
+    cy.intercept('GET', '/api/github-repositories?*', { fixture: 'github-repos.json' });
```

Comment:
This is super nice

Reply:
@Betree : Yes, it is, there's more new features we could leverage I think (such as retries) but I just tried to to the basic stuff so we have a workable solution. 😄